packages/next/src/server/config-shared.ts

```diff
@@ -100,15 +100,15 @@ export interface WebpackConfigContext {
   /** Number of total Next.js pages */
   totalPages: number
   /** The webpack configuration */
-  webpack: any
+  webpack: typeof webpack
```

Comment:
From line 2:

```ts
import type { webpack } from 'next/dist/compiled/webpack/webpack'
```

Since `webpack` is being imported as a type already, do we really need to use `typeof` here?

Reply:
Yes